resolwe/composer.py

```diff
@@ -0,0 +1,50 @@
+"""Support for easier dynamic composition of type extensions."""
+import inspect
+
+
+class Composer:
+    """Support for easier dynamic composition of type extensions."""
+
+    def __init__(self):
+        """Construct new composer instance."""
+        self._extensions = {}
+
+    def _get_class_path(self, klass_or_instance):
+        """Return class path for a given class.
+
+        :param klass_or_instance: Class or instance of given class
+        :return: String containing the class path
+        """
+        if inspect.isclass(klass_or_instance):
+            klass = "{}.{}".format(
```

Comment:
It would be nicer to have a method for constructing class name and use that one here and bellow. Maybe consider using __qualname__ instead of __name__  since it handles nested classes properly. But this is not needed in this situation.